src/sqlfluff/core/parser/grammar/sequence.py

```diff
@@ -14,7 +15,6 @@
 )
 from sqlfluff.core.parser.grammar.conditional import Conditional
 from sqlfluff.core.parser.match_algorithms import (
-    resolve_bracket,
```

Comment:
With the removal of `resolve_bracket` here - I think this is the only real use of the method. We can probably strip it out entirely if it isn't used here anymore.

Reply:
...actually ignore me here. I've checked and it's still used in the greedy matching algorithm.

Reply:
I want to get to that point - but it'll take a refactor on Anything()